macros/src/types/mod.rs

```diff
@@ -18,6 +19,10 @@ pub(crate) fn struct_def(s: &ItemStruct) -> Result<DerivedTS> {
 
 fn type_def(attr: &StructAttr, ident: &Ident, fields: &Fields) -> Result<DerivedTS> {
     let name = attr.rename.clone().unwrap_or_else(|| to_ts_ident(ident));
+    if let Some(t_o) = &attr.type_override {
```

Comment:
I think it's better to call the variable `type_override` instead of `t_o`. These abbreviatons are usually bad and they tend to become a habit if allowed

Reply:
this shadows the module name, which I try to avoid. I just couldn't think of a better name for it 😅 
I'm open to suggestions

Reply:
Maybe just `ty_override` then

Reply:
I went with `attr_type_override` because it plays better with rust analyzer's parameter name hints